code/modules/client/preference_setup/general/02_body.dm

```diff
@@ -658,7 +658,7 @@ var/global/list/valid_bloodtypes = list("A+", "A-", "B+", "B-", "AB+", "AB-", "O
 /datum/category_item/player_setup_item/proc/ResetFacialHair()
 	var/decl/species/mob_species = get_species_by_key(pref.species)
 	var/decl/bodytype/B = mob_species.get_bodytype_by_name(pref.bodytype)
-	var/list/valid_facialhairstyles = mob_species.get_facial_hair_styles(B.associated_gender)
+	var/list/valid_facialhairstyles = mob_species.get_facial_hair_styles(B?.associated_gender)
```

Comment:
This doesn't make any sense, if `B` is null then that implies `pref.bodytype` isn't set.

Reply:
Well, yeah?.. If it null, then we will pick any hair anywaay.

Reply:
`pref.bodytype` should _never_ be null.

Reply:
It is null.

![изображение](https://user-images.githubusercontent.com/44920739/124402283-f51a1880-dd37-11eb-93a5-65073a33eac7.png)

Reply:
I'm not questioning that it's null, I'm saying that it being null is a fundamentally more significant issue than not getting the right facial hair and should be investigated/fixed rather than bandaiding over it.